app/views/vacancies/_show.html.slim

```diff
@@ -5,4 +5,4 @@
   = strip_tags(@vacancy.job_summary)
```

Comment:
Same here on line 1 (a div is rendered because of the do block). These can both be one liners without being in a block